src/super_gradients/common/plugins/deci_client.py

```diff
@@ -173,6 +173,14 @@ def register_experiment(self, name: str, model_name: str, resume: bool):
         :param name:        Name of the experiment to register
         :param model_name:  Name of the model architecture to connect the experiment to
         """
+        try:
+            self.lab_client.register_user_architecture(BodyRegisterUserArchitecture(architecture_name=model_name))
+        except ApiException as e:
+            if e.status == 422:
+                logger.debug(f"The model was already registered, or validation error: {e.body}")
+            else:
+                raise e
+
```

Comment:
Question: Is this how we want to handle the exceptions?
Do we want only debug? Do we want to cover more exceptions status? Any other important thing to think of?

Reply:
yes. this is ok